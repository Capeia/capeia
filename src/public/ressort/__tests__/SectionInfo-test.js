// @flow
/* eslint-env jest, jasmine */

import React from 'react'
import renderer from 'react-test-renderer'

describe('<SectionInfo>', () => {
  it('renders correctly', () => {
    jest.deepUnmock('../SectionInfo')
    const Component = require('../SectionInfo').default
    const section = {
      background: '',
      info: () => <div>foo</div>
    }
    const noop = () => {}
    const tree = renderer.create(<Component section={section} onClose={noop} />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
