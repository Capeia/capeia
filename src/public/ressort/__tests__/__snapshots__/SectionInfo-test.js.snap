// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SectionInfo> renders correctly 1`] = `
<div>
  <div
    className="overlay"
  />
  <div
    className="wrapper"
  >
    <div
      className={undefined}
    >
      <div
        className="Lead"
        style={undefined}
      >
        
      </div>
      <div
        className="container-fluid"
      >
        <div
          className="row"
        >
          <div
            className="contentContainer  col-lg-6 col-lg-offset-3 col-md-8"
          >
            <div
              className={undefined}
            >
              <div
                className="text"
              >
                <div>
                  foo
                </div>
              </div>
            </div>
          </div>
          <div
            className="sidebarContainer col-lg-3 col-md-4"
          >
            <aside
              className="sidebar"
            />
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
